build_npm.ts

```diff
@@ -0,0 +1,68 @@
+import { build, emptyDir } from "https://deno.land/x/dnt/mod.ts";
+
+const version = Deno.args[0];
+
+if (!version) {
+  throw new Error("Please specify a version.");
+}
+
+await emptyDir("./npm");
+
+await build({
+  entryPoints: ["./mod.ts"], // Replace with your actual entry point
+  outDir: "./npm",
+  shims: {
+    deno: {
+      test: "dev",
+    },
+    custom: [
+      {
+        package: {
+          name: "web-streams-polyfill",
```

Comment:
Question: Can we just import these from [`node:stream/web`](https://nodejs.org/api/webstreams.html) which is built into Node.js? That would keep the overall npm package size smaller, which is a goal for this project (small and no third party dependencies outside std lib).

Reply:
it seems [node:stream/web](https://nodejs.org/api/webstreams.html) doesn't implement stream the same way:
![CleanShot 2024-02-16 at 17 37 12@2x](https://github.com/bradenmacdonald/deno-s3-lite-client/assets/4084527/4950e462-57f0-48c1-8098-b73bfd4cd002)

Reply:
Hmm, it might be just a type error; I wonder if it works if the type checking is disabled. But in any case, seems like it's not a straightforward change. Thanks for checking into it anyways.

Reply:
i will try to run integration test

Reply:
I managed to run integration test in Node.js and this doesn't work, it's not only the type:
![CleanShot 2024-02-17 at 00 41 16@2x](https://github.com/bradenmacdonald/deno-s3-lite-client/assets/4084527/294a1629-1a2b-4c06-b6b8-be123c2a5155)

Reply:
Ok in fact integration test fail as well with polyfill